src/components/GlobeOptimized.jsx

```diff
@@ -440,6 +523,8 @@ export default function GlobeOptimized(props) {
     );
     canvas.on('click', clicked);
 
+    canvas.on('mousemove', havoured);
```

Comment:
I think it's still triggered by `click` event instead of `hover` event, right? @sidgan

Reply:
star name should appear when mouse hovers over star. the only click functionality is for the meteors to show up additional information that takes time for a reader to consume and hence stays for a longer duration. 

Similarly constellation name should appear on a mouse hover over the lines that connect two stars forming a constellation.